packages/components/icons/src/index.ts

```diff
@@ -55,6 +55,7 @@ export { Download as DownloadIcon } from './Download';
 export { DownloadTrimmed as DownloadTrimmedIcon } from './DownloadTrimmed';
 export { Drag as DragIcon } from './Drag';
 export { DragTrimmed as DragTrimmedIcon } from './DragTrimmed';
+export { Diamond as Diamond } from './Diamond';
```

Comment:
Following the usage?

```suggestion
export { Diamond as DiamondIcon } from './Diamond';
```